security/compliance_operator/compliance-operator-release-notes.adoc

```diff
@@ -0,0 +1,75 @@
+
+
+[id="openshift-compliance-operator-0-1-44-bug-fixes"]
+=== Bug fixes
+
+* Previously, while performing a scan, an unexpected termination was occurring in one of the scanner containers of the pods. In this release, the Compliance Operator uses the latest OpenSCAP version 1.3.5 to avoid a crash.
++
+* Previously when you used `autoReplyRemediations` to apply remediations, this would trigger an update of the cluster nodes. This could be disruptive if some of the remediations did not include all of the required input variables. Now, if a remediation  is missing one or more required input variables, it is  assigned a state of `NeedsReview`.  If one or more remediations are in a `NeedsReview` state, the machine config pool will remain paused and the remediations will not be applied until all of the required variables are set. This helps minimize disruption to the nodes.
++
+* The RBAC Role and RoleBinding used for Prometheus metrics are changed to 'ClusterRole' and 'ClusterRoleBinding' to ensure that monitoring works out-of-the-box.
++
+* Previously, if an error occurred while parsing a profile, rules or variables objects would be removed and deleted from the profile. Now, if an error occurs during parsing, the `profileparser` annotates the object with a temporary annotation that prevents the object from being deleted until after parsing completes. link:https://bugzilla.redhat.com/show_bug.cgi?id=1988259[(BZ#1988259)].
++
+* Previously, an error occurred if titles or descriptions were missing from a tailored profile. Because the XCCDF standard requires titles and descriptions for tailored profiles, titles and descriptions are now required to be set in `TailoredProfile` CRs.
++ 
+* Previously, when using tailored profiles, `TailoredProfile` variable values were only allowed to be set using a specific selection set. This restriction is now removed, and `TailoredProfile` variables can be set to any value.
+
+[id="compliance-operator-release-notes-0-1-39"]
+== Release Notes for Compliance Operator 0.1.39
+The following advisory is available for the OpenShift Compliance Operator 0.1.39:
+
+* link:https://access.redhat.com/errata/RHBA-2021:3214[RHBA-2021:3214 OpenShift Compliance Operator Bug Fix and Enhancement Update]
+
+[id="compliance-operator-0-1-39-new-features-and-enhancements"]
+=== New features and enhancements
+
+* Previously, the Compliance Operator was unable to parse Payment Card Industry Data Security Standard (PCI DSS) references. Now, the Operator can parse compliance content that ships with PCI DSS profiles.
++
+* Previously, the Compliance Operator was unable to execute rules for AU-5 control in the moderate profile. Now, permission is added to the Operator so that it can read *Prometheusrules.monitoring.coreos.com* objects and execute the rules that cover AU-5 control in the moderate profile.
```

Comment:
```suggestion
* Previously, the Compliance Operator was unable to execute rules for AU-5 control in the moderate profile. Now, permission is added to the Operator so that it can read *Prometheusrules.monitoring.coreos.com* objects and run the rules that cover AU-5 control in the moderate profile.
```